src/components/chatbox/chatbox.module.css

```diff
@@ -94,13 +104,12 @@
 	resize: none;
 	background: transparent;
 }
-.button {
-	height: 24px;
-	width: 24px;
-	background: transparent;
-	&:focus {
-		outline: auto;
-	}
+
+.buttonContainer {
+	height: 42px;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
```

Comment:
The style changes here address [✓ Global: Please adjust the vertical alignment of the "send" cta- appears to be pushed down a few pixels- should align with the icon on the left of the input. ](https://app.asana.com/0/1100423001970639/1205523765153656)